### PR TITLE
Provider-agnostic authentication

### PR DESCRIPTION
**Pull request by johntron**
Because Blitz now uses UAA as well as third-party identity providers (Google, Facebook), the profile objects returned by api/1/account/about can vary, and this results in failed authentication for users without an email address. This PR fixes logging in and starting blitz/rushes for all identity providers by using the entire profile object rather than individual fields like email.

Also included:
* Increases popup width 5px to 485px so 'run' button isn't placed on a new line when rush is selected
* Fixes bug where rush options are hidden eventhough rush is selected - happens when selecting rush, then reopening the popup
* Fixes links to results

### CODE REVIEW

**Comment by johntron** [comment]
@leonoraEAE this is ready for review. All the bugs have been fixed, so we just need to merge this in and then publish the plugin.

File: blitz.chrome.js [hunk 1]
```diff
@@ -243,14 +255,14 @@ $(function () {
...
-                    blitz.abort(user_id, api_key, job._id);
+                    blitz.abort(profile, api_key, job._id);
                     $(this).remove();
                 });
                 test.results = [];
                 test.$progress.attr('value', '0');
             }
 
-            if (job.result && job.result.timeline) {
+            if (job.result && job.result.timeline && job.result.timeline.length) {
```
Comment (leonoraEAE) [inline blitz.chrome.js]: Probably would be best to test job.result.timeline.length > 0 given that a subtraction is being done on the line below.

**Comment by leonoraEAE** [comment]
Review complete. Just a small comment.

File: blitz.chrome.js (hunk 1 above)
Comment (johntron) [inline blitz.chrome.js]: The "> 0" comparison wouldn't add any more protection:

.length | truthiness | "> 0"
--------|---------------|-------
undefined | F | F
0 | F | F
1 | T | T

Comment (leonoraEAE) [inline blitz.chrome.js]: you're right. Ignore me. Sorry for the overhead.

**Pull request merged**